messaging/rabbitmq_binding.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+module: rabbitmq_binding
+author: Manuel Sousa
+version_added: 2.0
+
+short_description: This module manages rabbitMQ bindings
+description:
+  - This module uses rabbitMQ Rest API to create/delete bindings
+requirements: [ python requests ]
+options:
+    state:
+        description:
+            - Whether the exchange should be present or absent
+            - Only present implemented atm
+        choices: [ "present", "absent" ]
+        required: false
+        default: present
+    name:
+        description:
+            - source exchange to create binding on
+        required: true
+        aliases: [ "src", "source" ]
+    login_user:
+        description:
+            - rabbitMQ user for connection
+        required: false
+        default: guest
+    login_password:
+        description:
+            - rabbitMQ password for connection
+        required: false
+        default: false
+    login_host:
+        description:
+            - rabbitMQ host for connection
+        required: false
+        default: localhost
+    login_port:
+        description:
+            - rabbitMQ management api port
+        required: false
+        default: 15672
+    vhost:
+        description:
+            - rabbitMQ virtual host
+            - default vhost is /
+        required: false
+        default: "/"
+    destination:
+        description:
+            - destination exchange or queue for the binding
+        required: true
+        aliases: [ "dst", "dest" ]
+    destinationType:
+        description:
+            - Either queue or exchange
+        required: true
+        choices: [ "queue", "exchange" ]
+        aliases: [ "type", "destType" ]
+    routingKey:
+        description:
+            - routing key for the binding
+            - default is #
+        required: false
+        default: "#"
+    arguments:
+        description:
+            - extra arguments for exchange. If defined this argument is a key/value dictionary
+        required: false        
+'''
+
+EXAMPLES = '''
+# Bind myQueue to directExchange with routing key info
+- rabbitmq_binding: name=directExchange destination=myQueue type=queue routingKey=info
+
+# Bind directExchange to topicExchange with routing key *.info
+- rabbitmq_binding: name=topicExchange destination=topicExchange type=exchange routingKey="*.info"
+'''
+
+import requests
+import urllib
+import json
+
+def main():
+    module = AnsibleModule(
+        argument_spec = dict(
+            state = dict(default='present', choices=['present', 'absent'], type='str'),
+            name = dict(required=True, aliases=[ "src", "source" ], type='str'),
+            login_user = dict(default='guest', type='str'),
+            login_password = dict(default='guest', type='str'),
```

Comment:
logging sensible data can be turned off by adding a `no_log=True`. I would generally recommend it for passwords. 
~~~
login_password = dict(default='guest', type='str', no_log=True),
~~~